data/templates/esx.rackhdcallback

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# esx       callback to rackhd post installation API hook
+# description: calls back to rackhd post installation API hook
+
+echo "Attempting to call back to RackHD ESX installer"
+
+# *sigh*, busybox shell does not support {1..30}. Retry 30 times with 1 second
+# sleep in between.
+for retry in $(awk 'BEGIN { for ( i=0; i<30; i++ ) { print i; } }');
+do
+    wget -s http://<%=server%>:<%=port%>/api/common/templates/<%=completionUri%>
```

Comment:
@benbp : have you verified this design on 1.2.0 branch. I ask this question because I found this URl will not success.
It will always fail due to this code: https://github.com/RackHD/on-http/blob/master/lib%2Fapi%2F1.1%2Fsouthbound%2Ftemplates.js#L46

The root cause is that the static IP that assigned during OS installation has not been inserted into the lookup table. The ARP poller may solve this problem, but the 1.2.0 is still monitoring the isc-dhcp-leased result. Apparently the static IP is not assigned by isc-dhcp-server, so the lookup entry will not contain such entry.

So we may have to change to another URL to ensure such pure notification works, I tried the static file that in the repository, but the static URL will not publish to the message queue of HTTP response.

Maybe it's the good time to design some new API to do such notification work, but seems this is not a quick fix for 1.2.0 release.

Any better thought?